### ISSUE TITLE
Livedo racemosa

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6466]
**Preferred term label:**
Livedo racemosa

**Synonyms**

**Definition (free text, please give PubMed ID)**
Livedo racemosa describes a reddish-blue mottling of the skin in an irregular, reticular pattern. It differs from the more common livedo reticularis by its shape. Livedo racemosa consists of broken circular segments resulting in a seemingly larger pattern, as opposed to the fine, regular, complete network of livedo reticularis. Livedo racemosa results from permanent impairment of peripheral blood flow and, unlike livedo reticularis, it persists on warming.

PMID:26500860   PMID:31753828 

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0011276

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:615688   Vasculitis, autoinflammation, immunodeficiency, and hematologic defects syndrome
PMID: 27059682   11/15

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
